tests/e2e/scripts/execute.sh

```diff
@@ -51,6 +53,16 @@ kubectl create clusterrolebinding system:anonymous --clusterrole=cluster-admin -
 
 bash -x ${E2E_DIR}/scripts/fast_test.sh $1
 
+#failed cases number
+fail=`grep -e "SUCCESS\!" -e "FAIL\!" /tmp/testcase.log | awk '{print $6}' | sed -r "s/\x1B\[([0-9]{1,2}(;[0-9]{1,2})?)?[mGK]//g" | awk '{sum+=$1} END {print sum}'`
+
+#test Edge Autonomy
+#only when the above are all successful, we need to test Edge Autonomy function.
+if [ "$fail" == "0" ]
+  then
+    bash ${E2E_DIR}/scripts/autonomy.sh
+fi
+
```

Comment:
It's better to write this whole logic in deployment_test.go, we can exec shell command in go code.

Reply:
got it.